src/Tools/dotnet-monitor/CollectionRules/Actions/CollectionRuleActionConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Actions
+{
+    internal class CollectionRuleActionConstants
+    {
+        public const string EgressPathOutputValueName = "EgressPath";
```

Comment:
I had mentioned this to Justin in the PR for `CollectDump`, but I wanted to reuse this variable between all the actions. If we want it to live somewhere else, we could do that, but I figured we didn't want everyone either 1) calling into the `CollectDump` action to get its value, or 2) copy-pasting this variable for each action.